{"issue_type": "request_error", "error_msg": "HTTPConnectionPool(host='www.ed.gov', port=80): Max retries exceeded with url: /robots.txt (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f1dade47f40>: Failed to establish a new connection: [Errno 101] Network is unreachable'))"}